nrt/data/TrapezoidalTest/Test.jobs

```diff
@@ -22,6 +22,7 @@
     </dyn:modeler>
     <dyn:simulation startTime="0" stopTime="10"/>
     <dyn:outputs directory="outputs">
+      <dyn:curves inputFile="Test.crv" exportMode="CSV"/>
```

Comment:
Is it on purpose ?

Reply:
Adrien asked me to add a reference for TrapezoidalTest.

Reply:
Ok, thanks